bit_manipulation/swapping_two_numbers.cpp

```diff
@@ -0,0 +1,88 @@
+/**
+ * @file
+ * @brief Swapping two numbers using Bitwise operator
+ * More about it: https://betterexplained.com/articles/swap-two-variables-using-xor
```

Comment:
Markdown link formatting should be used here.
```suggestion
 * @brief [Swap two numbers using Bitwise Operator](https://betterexplained.com/articles/swap-two-variables-using-xor) implementation
```